src/org/scada_lts/dao/pointvalues/PointValueAdnnotationsDAO.java

```diff
@@ -66,8 +67,18 @@
 				+COLUMN_NAME_SOURCE_ID + " "
 			+ "from "
 				+"pointValueAnnotations ";
-	
-	
+	private static final String PREFIX_NAME_FOR_POINTVALUEADNNOTATION_TABLE = "pv.";
```

Comment:
These names are long for me.

Reply:
These names or this name? For me no. I read this code and I know what this variable means.Please suggest another one.

Reply:
These are the suggestions
At the docker you can see that the task has been completed: (Modbus is not tested)
![Screenshot from 2019-11-15 07-48-42](https://user-images.githubusercontent.com/1280575/68924854-f624bd80-0781-11ea-8ab5-ad8b80a44120.png)